README.md

```diff
@@ -78,6 +78,7 @@ This library is used in the following projects:
 
 * MQTT Client Rx (Wrapper for Reactive Extensions, <https://github.com/1iveowl/MQTTClient.rx>)
 * Wirehome (Open Source Home Automation system for .NET, <https://github.com/chkr1011/Wirehome>)
+* MQTT Tester app (for [Android](https://play.google.com/store/apps/details?id=com.liveowl.mqtttester), [iOS](https://itunes.apple.com/us/app/mqtt-tester/id1278621826?mt=8), and [Windows 10](https://www.microsoft.com/en-us/store/p/mqtt-tester/9wzdncrd272c))
```

Comment:
I forgot to mention that it is ordered alphabetically 😄 So please just move it up one line. And I have one suggestion. Move the app into the "description" part. So that it looks like "MQTT Tester (MQTT client test app for [Android]( ..."